pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -192,6 +198,38 @@ def create_advisory(self, type: str, kind: str, impetus: str) -> int:
         advisory_num = int(match[1])
         return advisory_num
 
+    def get_advisories(self) -> Dict[str, int]:
+        repo = self.working_dir / "ocp-build-data-push"
+        shutil.rmtree(repo, ignore_errors=True)
```

Comment:
better to remove the repo in `save_advisories` function as well.

Reply:
It was already doing it :)